compiler/rustc_typeck/src/astconv/mod.rs

```diff
@@ -688,7 +688,7 @@ impl<'o, 'tcx> dyn AstConv<'tcx> + 'o {
 
         self.ast_path_to_mono_trait_ref(
             trait_ref.path.span,
-            trait_ref.trait_def_id().unwrap_or_else(|| FatalError.raise()),
+            trait_ref.trait_def_id().unwrap(),
```

Comment:
I don't see any value in `unwrap_or_else(|| FatalError.raise())`, though I think this could be an `.expect` call if we want a better message.

Reply:
nvm, this is to suppress ICEs I guess